cmd/lora/main.go

```diff
@@ -115,7 +119,14 @@ func main() {
 		}, []string{"method"}),
 	)
 
-	go subscribeToLoRaBroker(svc, mqttConn, logger)
+	msub, err := mqttsub.NewSubscriber(cfg.loraMsgURL, cfg.subTimeout, logger)
```

Comment:
I would use `mqtt` without package alias

Reply:
I agree - no need for this alias as it it is not overlapping with anything.

Only argument might be that it is somehow documenting wight from the start the intention of what we want to do with it in this module, and somehow prevent some other programmer to use it for PUB (or change the alias at this moment). If that was the intention, then I can accept it.

@dusanb94 @nmarcetic what is your opinion on this?

Reply:
Agree, if there is no overlapping, no need for an alias.

Reply:
I really don't mind, both look fine with me.

Reply:
I was following how it is used in mqtt:
https://github.com/mainflux/mainflux/blob/master/cmd/mqtt/main.go#L24
but there is overlapping so it had to be with alias.

Since here there is no other mqtt package, I removed alias.